contracts/src/interfaces/IEigenDACertVerifier.sol

```diff
@@ -4,7 +4,16 @@ pragma solidity ^0.8.9;
 import {IEigenDAThresholdRegistry} from "./IEigenDAThresholdRegistry.sol";
 import "./IEigenDAStructs.sol";
 
-interface IEigenDACertVerifier {
+interface IEigenDACertVerifierBase {
+    /// @notice Check a DA cert's validity, and revert if invalid.
+    function verifyDACert(bytes calldata certBytes) external view;
```

Comment:
what's the rationale for having different entry-points - iirc our last spec discussion aligned on the usage of a single verification function

Reply:
Yes, sorry. It's been a while since I pushed an update. This is going to go away.